calico-enterprise/getting-started/install-on-clusters/kubernetes/helm.mdx

```diff
@@ -38,7 +33,227 @@ In this guide, you install the Tigera Calico operator and custom resource defini
+
+<Tabs>
+
+<TabItem label="Standalone" value="Standalone">
+
+To install a standard {{prodname}} cluster with Helm:
+
+1. [Configure a storage class for Calico Enterprise](../../../operations/logstorage/create-storage).
+
+1. Install the Tigera {{prodname}} operator and custom resource definitions using the Helm 3 chart:
+
+  ```bash
+  helm install {{prodnamedash}} projectcalico/tigera-operator --version {{releaseTitle}} \
+  --set-file imagePullSecrets.tigera-pull-secret=<path/to/pull/secret>,tigera-prometheus-operator.imagePullSecrets.tigera-pull-secret=<path/to/pull/secret> \
+  --set-file licenseKeyContent=<path/to/license/file/yaml> \
+  --namespace tigera-operator --create-namespace
+  ```
+
+  or if you created a `values.yaml` above:
+
+  ```bash
+  helm install {{prodnamedash}} projectcalico/tigera-operator --version {{releaseTitle}} -f values.yaml \
+  --set-file imagePullSecrets.tigera-pull-secret=<path/to/pull/secret>,tigera-prometheus-operator.imagePullSecrets.tigera-pull-secret=<path/to/pull/secret> \
+  --set-file licenseKeyContent=<path/to/license/file/yaml> \
+  --namespace tigera-operator --create-namespace
+  ```
+
+  :::note
+
+  The licenseKey must have `ApiVersion: crd.projectcalico.org/v1`.
+
+  :::
+
+1. You can now monitor progress with the following command:
+
+  ```bash
+  watch kubectl get tigerastatus
+  ```
+
+  Congratulations! You have now installed {{prodname}} using the Helm 3 chart.
+
+</TabItem>
+<TabItem label="Management" value="Management">
+
+To install a {{prodname}} [management](../../../multicluster/create-a-management-cluster#value) cluster with Helm:
+
+1. [Configure a storage class for Calico Enterprise](../../../operations/logstorage/create-storage).
+
+1. Export the service port number, and the public IP or host of the management cluster. (Ex. "example.com:1234" or "10.0.0.10:1234".)
+
+  ```bash
+  export MANAGEMENT_CLUSTER_ADDR=<your-management-cluster-addr>
+  ```
+
+1. Export the service type (`NodePort` or `LoadBalancer`), the service port, target port and protocol.
+  
+  ```bash
+  export EXT_SERVICE_TYPE=<external-service-type>
+  export EXT_SERVICE_PORT=<external-service-port>
+  export EXT_SERVICE_TARGET_PORT=<external-service-target-port>
+  export EXT_SERVICE_PROTOCOL=<external-service-protocol>
+  ```
+  
+  If the external service is of type NodePort.
+
+  ```bash
+  export EXT_SERVICE_TYPE=<external-service-node-port>
+  ```
+
+1. Export one or more managed clusters.
+
+  ```bash
+  export MANAGED_CLUSTER_NAME=<managed-cluster-name>
+  export MANAGED_CLUSTER_OPERATOR_NAMESPACE=<managed-cluster-operator-namespace>
+  export MANAGED_CLUSTER_CERTIFICATE=<managed-cluster-certificate>
+  ```
+
+1. Install the Tigera {{prodname}} operator and custom resource definitions using the Helm 3 chart, with a NodePort service:
+
+  ```bash
+  helm install {{prodnamedash}} projectcalico/tigera-operator --version {{releaseTitle}} -f values.yaml \
```

Comment:
Have you considered putting all the `--set` flags in a snippet in the way that it can be added to their values.yaml? I suppose this is how most people will be using it.

Perhaps even change it to:
```
echo " <all the options> " >> values.yaml

```

Reply:
I replaced the list of `--set` with an update to the values
The install command still contains a reference to the pull secret and license `--set-file`